opm/simulators/wells/MultisegmentWellAssemble.cpp

```diff
@@ -212,6 +216,12 @@ assembleAccelerationTerm(const int seg_target,
     // acceleration term shold be
     //  * velocity head for seg_target if seg = seg_target
     //  * negative velocity head for seg if seg != seg_target   
+    
+    /*
+        This method is called in MultisegmentWellEval::assembleAccelerationPressureLoss.
```

Comment:
We might want to remove some of these comments for development after the parallel msw features is done.

Reply:
I believe these comments are quite useful for understanding why the [sumDistributed](https://github.com/OPM/opm-simulators/blob/master/opm/simulators/wells/MultisegmentWell_impl.hpp#L1898) function is called at that specific point during the assembly process.
However, if you have strong reasons for removing them, I’m happy to do so.